### DISCUSSION TITLE
附录插入算法伪代码第一页为空白页

### DISCUSSION THREAD

**Discussion by Flora-ll** [note286/xduts#135] [Q&A] [answered]
在附录中插入算法伪代码第一页为空白页，两个算法全跑到了在第二页中，有什么办法解决吗？
导言区：
```latex
    info = {
        appendix={appendix.tex}
    }
```
appendix.tex：
```latex
\chapter{}
  \begin{algorithm}
	......
  \end{algorithmic}  
\end{algorithm}
```

编译出的PDF:
![附录问题](https://github.com/note286/xduts/assets/84665106/a7984dfa-69d6-4367-bd8d-2f7b931261b7)

**Comment by note286** [comment, marked as answer]
你试试给`algorithm`加上`[h]`。

**Reply by Flora-ll** [reply]
好的，谢谢大佬！回的好快呀感动

**Reply by note286** [reply]
我没有测试，我猜测是浮动体的问题，如果不好使你再问，我再实际测试一下。

**Reply by Flora-ll** [reply]
确实是的，加完之后有改善，不过还是不美观，我又设置了算法跨页就好多了。谢谢大佬！
